src/manhole/__init__.py

```diff
@@ -367,7 +367,14 @@ def handle_repl(locals):
     }
     if locals:
         namespace.update(locals)
-    ManholeConsole(namespace).interact()
+    try:
+        ManholeConsole(namespace).interact()
+    finally:
+        for attribute in ['last_type', 'last_value', 'last_traceback']:
+            try:
+                delattr(sys, attribute)
```

Comment:
Probably better to set to None instead? This would trigger AttributeErrors - afaik the python docs give the impression that these attributes would always be there.

Reply:
Ooof, actually I'm wrong https://docs.python.org/3/library/sys.html?highlight=last_traceback#sys.last_traceback

Reply:
Yes, they're not defined by default. So I'm reverting them to original state